lib/rest/Request.js

```diff
@@ -0,0 +1,207 @@
+"use strict";
+
+const Base = require("../structures/Base");
+const {fetch} = require("undici");
```

Comment:
Browser support is still around in some form:

```suggestion
const {fetch, FormData} = typeof window !== "undefined" ? window : require("undici");
```

(for the time being - once global `fetch` is stabilized in Node, then only the global should be used)